### PR TITLE
Version checking on bootstrap

### PR DESCRIPTION
**Pull request by darylhedley**
Adds upgrade script to the authoring tool

### CODE REVIEW

File: lib/application.js [hunk 1]
```diff
@@ -26,6 +26,9 @@ var EventEmitter = require('events').EventEmitter,
     Mailer = require('./mailer').Mailer,
     configuration = require('./configuration');
 
+var request = require('request');
+var async = require('async');
+var chalk = require('chalk');
```
Comment (dennis-learningpool) [inline lib/application.js]: The install.js uses 'colors' still. Chalk looks better. Can you update the install.js to swap out colors usage for chalk, and remove colors from the dependencies too?

pretty please :)

Comment (brian-learningpool) [inline lib/application.js]: I've removed 'colors'.

**Comment by brian-learningpool** [comment]
+1

**Comment by dennis-learningpool** [comment]
+1

**Pull request merged**